client/source/js/modules/project/open-ctrl.js

```diff
@@ -49,10 +53,14 @@ define(['./module', 'angular', 'underscore'], function (module, angular, _) {
      *
      * Alerts the user if it cannot do it.
      */
-    $scope.edit = function (name) {
-      $http.get('/api/project/open/' + name)
+    $scope.edit = function (name, id) {
+      $http.get('/api/project/open/' + id)
         .success(function (response) {
-          activeProject.setActiveProjectFor(name, UserManager.data);
+          if (response && response.status === 'NOK') {
+            alert(response.reason);
```

Comment:
Error handling … please also not use alert. We got rid of all the existing ones most recently